index.js

```diff
@@ -11,6 +11,21 @@ const changePrototype = (to, from) => {
 	Object.setPrototypeOf(to, fromPrototype);
 };
 
+const wrappedToString = (fromBody, name) => {
+	const withToName = name === '' ? '' : `with ${name}() `;
+	return `/* Wrapped ${withToName}*/\n${fromBody}`;
+};
+
+const changeToString = (to, from, name) => {
+	const value = wrappedToString.bind(null, from.toString(), name.trim());
```

Comment:
We trim `name` because it's not uncommon to have function names with trailing whitespaces.

```
> (() => {}).bind().name
'bound '
```